util/string.go

```diff
@@ -2,9 +2,9 @@ package util
 
 import "strings"
 
-// Truncate takes a string and truncates it
-func Truncate(str string, length int) string {
-	if len(str) < length {
+// Truncate takes a string and truncates it (if doTruncate is true)
+func Truncate(str string, length int, doTruncate bool) string {
```

Comment:
A bit weird to give a function a true/false boolean on wether or not to do what its name says it will do.

But I see how it means less conditionals in `list_keypairs.go`

Reply:
Right. Doesn't look nice, but helps keep the output code readable.